src/components/Main.vue

```diff
@@ -86,18 +86,15 @@ export default {
     },
 
     methods: {
+
         highlightIssue(issue){
             this.highlightStr = issue;
         },
+
         Again() {
             this.rendered = false; 
         },
```

Comment:
Add a verb to this method name: `doAgain`? `clickAgain`? If it's an event handler, it should be `onClickAgain`.